libraries/chain/backing_store/tests/benchmark_kv.cpp

```diff
@@ -0,0 +1,337 @@
+#include <sys/resource.h>
+
+#include <eosio/chain/kv_chainbase_objects.hpp>
+#include <eosio/chain/backing_store/kv_context_rocksdb.hpp>
+#include <eosio/chain/backing_store/kv_context_chainbase.hpp>
+
+#include <boost/filesystem.hpp>
+
+using namespace eosio;
+using namespace eosio::chain;
+
+namespace kv_benchmark {
+
+// Global test data
+constexpr account_name receiver = N(kvrdb);
+constexpr uint64_t contract = receiver.to_uint64_t();
+constexpr account_name payer = N(payer);
+std::string prefix = "prefix";
+constexpr uint32_t it_key_size = 10;
+constexpr uint64_t default_billable_size = 12;
+
+constexpr uint32_t default_value_size = 1024;
+std::string default_operation = "get";
+struct cmd_args {
+   uint32_t num_keys = 0;
+   uint32_t value_size = default_value_size;
+   bool is_rocksdb = true; // false means chainbase
+   std::string operation = default_operation;
+   std::string key_file = "";
+};
+
+constexpr uint32_t work_set_size = 10000;
+constexpr uint32_t num_iterations = 100;
```

Comment:
Why is this so small?  As I mentioned elsewhere, this should be sweepable so that we can see when things fall over.

Reply:
I will remove std::chrono::steady_clock::now. It is unnecessary. work_set_size has already been removed; it is now calculated from the set itself. num_iterations is now a parameter, with default value 100000.